satpy/etc/enhancements/generic.yaml

```diff
@@ -70,6 +70,16 @@ enhancements:
         stretch: crude
         min_stretch: [-25, -40, 243]
         max_stretch: [0, 5, 208]
+  airmass_abi_default:
```

Comment:
Could you remove the `_default` here? People are starting to think it means something when really I was just using `_default` to mean "not specific to any sensor or other specific identifying feature". @mraspaud not sure if you have an opinion on this.